.doc_gen/readmes/README.md

```diff
@@ -20,9 +20,10 @@ and install packages by running the following commands in the
 `.doc_gen/readmes` folder:
 
 ```
-python -m venv venv
-venv/Scripts/activate
-python -m pip install -r requirements.txt
+python -m venv .venv
```

Comment:
Also will need to add .venv to the .gitignore file in this folder.

Reply:
Done